ga4gh/cli.py

```diff
@@ -816,3 +820,72 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+class TestConfigExists(unittest.TestCase):
+    """This class implements configuration checks for the GA4GH server. It is
+    structured as a set of unit tests, and relies on the unittest module.
+    However, as some of the tests require state injection (through command
+    line arguments) we need to break the usual unittest pattern and instead
+    implement our own simple test runner.
+    """
+
+    def __init__(self, parser=None):
+        self.parser = parser
+
+    def setUp(self):
+        self.app = flask.Flask(__name__)
+        parser = self.parser if self.parser is not None \
```

Comment:
@jeromekelleher claims that line-continuation backslashes kill puppies, so I would avoid this

I don't know if we have an opinion on ternary operators.  Personally, I avoid them, but I'd be willing to hear what other people think...